test_pull_requests

```diff
@@ -359,6 +361,78 @@ def compose_bot_comment(prefix, comment_data={})
   updated_comment
 end
 
+class FlushableStore
+  def initialize(c_file=nil)
+    @cache_file = nil
+    @cache = {}
+    if c_file
+      @cache_file = File.expand_path(c_file)
+      if File.exists? @cache_file
```

Comment:
> I don't want the script to die on load failure

Why?

If I ask for a cache and we can't make it happen, why should we continue? That  seems like a critical failure to init the system.